### PR TITLE
Add basic descriptions to YAML pipelines

### PR DESCRIPTION
**Pull request by tcare**
Added some basic descriptions and did some drive-by whitespace cleanup.

### CODE REVIEW

File: .pipelines/azdo-base-pipeline.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Pipeline template to run linting, unit tests, code coverage, and publish the results.
```
Comment (jotaylo) [inline .pipelines/azdo-base-pipeline.yml]: >, [](start = 46, length = 1)

unit tests with code coverage

File: .pipelines/azdo-pr-build-train.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# PR pipeline for running CI on PRs to master.
```
Comment (jotaylo) [inline .pipelines/azdo-pr-build-train.yml]: >CI on PRs [](start = 26, length = 9)

Pipeline to run basic code quality tests as part of Pull Requests to master

File: .pipelines/diabetes_regression-ci-build-train.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# CI pipeline that builds, trains, tests, and deploys the diabetes model.
```
Comment (jotaylo) [inline .pipelines/diabetes_regression-ci-build-train.yml]: > CI [](start = 1, length = 3)

Continuous Integration (CI) pipeline

Comment (jotaylo) [inline .pipelines/diabetes_regression-ci-build-train.yml]: >diabetes [](start = 58, length = 8)

diabetes_regression (bootstrap script should update this to the project name)

Comment (tcare) [reply .pipelines/diabetes_regression-ci-build-train.yml]: That file is already covered in bootstrap, sounds good.

Comment (jotaylo) [inline .pipelines/diabetes_regression-ci-build-train.yml]: >builds [](start = 19, length = 6)

Continuous Integration (CI) pipeline that orchestrates the training, evaluation, registration, deployment, and testing of the diabetes_regression model.

Comment (tcare) [reply .pipelines/diabetes_regression-ci-build-train.yml]: On that note, I think that this file could be named better (maybe just -ci) but I think that renaming the file could potentially cause issues. The same for 'base' which is really more targeted towards code quality, though could still exist for the purpose of being included in most/all pipelines.

**Comment by jotaylo** [comment]
> 

Can you add comments for these

---
Refers to: .pipelines/diabetes_regression-variables.yml:54 in 9e60ab4. [](commit_id = 9e60ab47c8552b05e18aff656db1843a2ff1878e, deletion_comment = False)

**Comment by tcare** [comment]
> > 
> 
> Can you add comments for these
> 
> Refers to: .pipelines/diabetes_regression-variables.yml:54 in [9e60ab4](https://github.com/microsoft/MLOpsPython/commit/9e60ab47c8552b05e18aff656db1843a2ff1878e). [](commit_id = [9e60ab4](https://github.com/microsoft/MLOpsPython/commit/9e60ab47c8552b05e18aff656db1843a2ff1878e), deletion_comment = False)

Spoke offline, there needs to be a larger cleanup here and is out of scope for this PR.

File: .pipelines/diabetes_regression-ci-build-train.yml (hunk 3 above)
Comment (jotaylo) [reply .pipelines/diabetes_regression-ci-build-train.yml]: Agree. For this file I'd suggest -ci-train-deploy. "-code-quality" sounds good for "base".

Comment (tcare) [reply .pipelines/diabetes_regression-ci-build-train.yml]: I'll create an item for this as I want to understand the implications of renaming w.r.t. compat.

**Comment by tcare** [comment]
Oh, I was going to merge, but thanks :)

**Pull request merged**